packages/jwt-middleware/README.md

```diff
@@ -9,23 +9,40 @@ This middleware allows to secure execution of a webtask using JWT tokens by vali
 ```
 {
     "@webtask/middleware-compiler": "1.3.0", 
-    "express-jwt": "5.3.1",
-    "jws": "3.1.4",
-    "jwks-rsa": "1.2.1"
+    "@webtask/jwt-middleware": "1.0.0"
 }
 ```
 
 2. Set the `wt-compiler` metadata property on your webtask to `@webtask/middleware-compiler`.
 
-3. Set the `wt-compiler` metadata property to `@webtask/jwt-middleware`.
+3. Set the `wt-middleware` metadata property to `@webtask/jwt-middleware`.
 
-4. Set the `wt-authorize-execution` metadata property to any value other than `0` to secure the webtask.  Value of `0` or missing `wt-authorize-execution` metadata property will result with the webtask execution being unsecure.
+4. Set the `wt-authorize-execution` metadata property to any value other than `0` to secure the webtask.  Value of `0` or missing `wt-authorize-execution` metadata property will result in the webtask execution being unsecure.
 
-5. Set the `iss` metadata property to the value of `authorization_server` property obtained from the discovery endpoint of your deployment (located at `{deployment_url}//api/description`).
+5. Set the `wt-execution-iss` metadata property to the value of `authorization_server` property obtained from the discovery endpoint of your deployment (located at `{deployment_url}//api/description`).
```

Comment:
Seems like there is an extra `/` in both (5) and (6).

Reply:
Fixed and checked in.